src/main/java/com/google/sps/servlets/LoadGameServlet.java

```diff
@@ -40,19 +40,27 @@
 public class LoadGameServlet extends HttpServlet {
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        // Currently ignores the input and just returns a fixed Game.
-        String gameID = "demogameid";
+        String gameID = getGameID(request);
```

Comment:
In my opinion, I think it's clearer to have a separate function for each individual parameter. This is because some parameters are formatted differently than others (Strings, ints, arrays, etc.), so I need to parse them in different ways. If I parse them all together in one function, each in its own way, things could get confusing.